spec/services/zendesk_diy_intake_service_spec.rb

```diff
@@ -90,17 +90,22 @@
       expect(body).to include "send_diy_confirmation"
     end
 
-    context "with corresponding full service tickets" do
+    context "with corresponding intake tickets" do
       let(:intake_ticket_map) do
         { 99998 => double(ZendeskAPI::Ticket, id: "99998"),
           99997 => double(ZendeskAPI::Ticket, id: "99997") }
       end
       let!(:related_intakes) do
         intake_ticket_map.map do |ticket_id, _|
-          create(
+          ticket_id.even? ? create(
```

Comment:
This is fine, but a little excessively slick IMHO :)

No action required

Reply:
i was being lazy :(

Reply:
lazy and slick are sorta synonyms; it's not a showstopper